src/main/java/mServer/crawler/sender/phoenix/tasks/PhoenixOverviewTask.java

```diff
@@ -0,0 +1,76 @@
+package mServer.crawler.sender.phoenix.tasks;
+
+import com.google.gson.reflect.TypeToken;
+import de.mediathekview.mlib.Config;
+import java.lang.reflect.Type;
+import java.util.Collection;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import javax.ws.rs.client.WebTarget;
+import mServer.crawler.sender.MediathekReader;
+import mServer.crawler.sender.orf.CrawlerUrlDTO;
+import mServer.crawler.sender.orf.tasks.AbstractRecursivConverterTask;
+import mServer.crawler.sender.phoenix.SendungOverviewDto;
+import mServer.crawler.sender.phoenix.parser.PhoenixSendungOverviewDeserializer;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class PhoenixOverviewTask extends ZdfTaskBase<CrawlerUrlDTO, CrawlerUrlDTO> {
+
+  private static final Logger LOG = LogManager.getLogger(PhoenixOverviewTask.class);
+
+  private static final Type OPTIONAL_OVERVIEW_DTO_TYPE_TOKEN = new TypeToken<Optional<SendungOverviewDto>>() {
+  }.getType();
+
+  private final String baseUrl;
+  private final int subpage;
+
+  public PhoenixOverviewTask(MediathekReader aCrawler,
+          ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDtos, Optional<String> aAuthKey, String aBaseUrl) {
+    this(aCrawler, aUrlToCrawlDtos, aAuthKey, aBaseUrl, 0);
+  }
+
+  public PhoenixOverviewTask(MediathekReader aCrawler,
+          ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDtos, Optional<String> aAuthKey, String aBaseUrl, int aSubpage) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDtos" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)